pylint/checkers/imports.py

```diff
@@ -850,6 +859,88 @@ def _check_imports_order(
                         )
         return std_imports, external_imports, local_imports
 
+    def _get_out_of_order_string(
+        self,
+        third_party_imports: list[tuple[ImportNode, str]] | None,
+        first_party_imports: list[tuple[ImportNode, str]] | None,
+        local_imports: list[tuple[ImportNode, str]] | None,
+    ) -> str:
+        # construct the string listing out of order imports used in the message
+        # for wrong-import-order
+        if third_party_imports:
+            plural = "s" if len(third_party_imports) > 1 else ""
+            imports_list = ", ".join(
+                [f'"{self._get_full_import_name(tpi)}"' for tpi in third_party_imports]
+            )
+            third_party = f"third party import{plural} {imports_list}"
```

Comment:
```suggestion
            plural = "s" if len(third_party_imports) > 1 else ""
            if len(third_party_imports) > MAX_NUMBER_OF_IMPORT_SHOWN:
                        imports_list = ", ".join(
                            [f'"{self._get_full_import_name(tpi)}"' for tpi in third_party_imports[:MAX_NUMBER_OF_IMPORT_SHOWN/2]
                        } + " (...) " +", ".join(
                            [f'"{self._get_full_import_name(tpi)}"' for tpi in third_party_imports[-MAX_NUMBER_OF_IMPORT_SHOWN/2:]
                        }
            else:
                imports_list = ", ".join(
                    [f'"{self._get_full_import_name(tpi)}"' for tpi in third_party_imports]
                )
            third_party = f"third party import{plural} {imports_list}"
```
What do you think ? This would permit to not have long message when there's a lot of import.

Reply:
I think that is a good idea, because it should provide enough information for the developer to fix things without the possibility of having 20 lines of imports to sort through.